OpenRA.Mods.Common/Traits/Attack/AttackSuicides.cs

```diff
@@ -14,30 +14,31 @@
 using OpenRA.Mods.Common;
 using OpenRA.Mods.Common.Activities;
 using OpenRA.Mods.Common.Orders;
-using OpenRA.Mods.Common.Traits;
 using OpenRA.Traits;
 
-namespace OpenRA.Mods.RA.Traits
+namespace OpenRA.Mods.Common.Traits
 {
-	class DemoTruckInfo : ITraitInfo, Requires<ExplodesInfo>
+	[Desc("Does a suicide attack where it moves next to the target when used in combination with `Explodes`.")]
+	class AttackSuicidesInfo : ITraitInfo
```

Comment:
This needs a `Requires<IMoveInfo>` because you use `Trait` (no `OrDefault`) below.